curtains/curtains-proguard.pro

```diff
@@ -0,0 +1,8 @@
+-keepclassmembernames class androidx.appcompat.view.WindowCallbackWrapper {
+    android.view.Window$Callback mWrapped;
```

Comment:
just keeping the member name. would it be safer to keep the field, instead of simply avoid the obfuscation of the name?

Reply:
Probably, as we definitely need to field to be there to be able to invoke it by reflection.

Reply:
ok. just changed with a simple `keep`